main.py

```diff
@@ -119,14 +119,16 @@ def get_current_hand(round_one_hand, round_two_hand):
     print(f'KoS blk: {kos_blk}')
     print(f'KoS mod: {kos_mod}')
     print(f'Kos mod msg: {kos_mod_message}')
-    print(f'\nPlayer HP: {player_hp}')
+    print(f'\nPlayer HP: {player.hp}')
     print(f'Kos HP: {kos_hp}')
 
 
+
 print("\n===== GAME OVER =====")
-if player_hp <= 0 and kos_hp > 0:
-    print("\nYou lost. Lose one reputation rank, and you are now injured.")
-if kos_hp <= 0 and player_hp > 0:
+if not player.is_alive:
+    if kos_is_alive:
+        print("\nYou lost. Lose one reputation rank, and you are now injured.")
+    if not kos_is_alive <= 0:
```

Comment:
Should be kos_hp or "False" or something

Reply:
Ah! Very good catch. 👀